app/controllers/hyrax/dashboard/collections_controller_decorator.rb

```diff
@@ -28,10 +28,12 @@ def update_valkyrie_collection
         result = transactions['change_set.update_collection']
                  .with_step_args(
                           'collection_resource.save_collection_banner' => { update_banner_file_ids: params["banner_files"],
-                                                                            alttext: params["banner_text"]&.first },
+                                                                            alttext: params["banner_text"]&.first,
+                                                                            banner_unchanged_indicator: params["banner_unchanged"] },
                           'collection_resource.save_collection_logo' => { update_logo_file_ids: params["logo_files"],
                                                                           alttext_values: params["alttext"],
-                                                                          linkurl_values: params["linkurl"] },
+                                                                          linkurl_values: params["linkurl"],
+                                                                          logo_unchanged_indicator: false },
```

Comment:
Why are we explicitly setting this to `false`?

Reply:
That is brought forward directly from Hyrax. I'm not sure why the form handles it differently, but it does.